backend/providers/aws/script_runner/services/ssm_executor.py

```
@@ -791,3 +791,78 @@ async def execute_script(
                 "output": "",
                 "error": str(e),
             }
+
+    def execute_script_batch_sync(
+        self,
+        instance_ids: List[str],
+        script_content: str,
+        interpreter: str,
+        account_id: str,
+        region: str,
+        credentials: Any,
```

Ruff (0.14.14)

Unused method argument: `credentials`

(ARG002)